ext/parquet/parquet.go

```diff
@@ -37,8 +40,40 @@ func (f *Format) NewDecoder(r io.Reader) (feedx.FormatDecoder, error) {
 }
 
 // NewEncoder implements Format.
-func (*Format) NewEncoder(w io.Writer) (feedx.FormatEncoder, error) {
-	return nil, fmt.Errorf("not implemented")
+func (f *Format) NewEncoder(w io.Writer) (feedx.FormatEncoder, error) {
+	// TODO! schema will need to be configured
+	// TODO! we could probably use the format Columns
+	schemaDef, err := parquetschema.ParseSchemaDefinition(
+		`message test {
+                       required int64 id;
+                       required binary city (STRING);
+                       optional int64 population;
+               }`)
+	if err != nil {
+		return nil, err
+	}
+
+	fw := goparquet.NewFileWriter(w,
+		goparquet.WithCompressionCodec(parquet.CompressionCodec_SNAPPY),
```

Comment:
I'm not sure you should make those decisions for the user, I suggest you simply add:

```
WriterOptions []goparquet.FileWriterOption
```
to `type Formatter` + some additional helpers:

```
With(opts ...goparquet.FileWriterOption) 
WithSchema(sd *parquetschema.SchemaDefinition)
```